src/main/java/com/ibm/cloud/sdk/core/service/BaseService.java

```diff
@@ -101,16 +101,24 @@ public BaseService(final String name, Authenticator authenticator) {
     }
     this.authenticator = authenticator;
 
+    // Configure a default client instance.
+    this.client = configureHttpClient();
```

Comment:
@padamstx - just to be sure, the client initiating should not be dependent on external config correct?

Reply:
Correct, "configureHttpClient()" does not involve external config stuff.